### PR TITLE
close atoum/atoum-documentation#157

### LINKED ISSUE #157: extension asserter : missing documentation
There is no documentation on the extension asserter.

It could be use like this : 

```
$this->extension('json')->isLoaded();
```

That will trigger an error if the json extension is not loaded.

We can achieve the same with the `@extensions` tag and the `--fail-if-skipped-methods`.

Is the extension asserter deprecated ? should it be documented ? // ping @atoum/all

### PR DESCRIPTION
**Pull request by Grummfy**
close #157

### CODE REVIEW

File: source/en/asserters/extension.inc.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.. code-block:: php
+
+	<?php
+	$this
+		->extension('json')
+		->isLoaded()
+		;
+
+.. note::
+	You can also use the tag :ref:`extension<annotation-php-extension>`, in some case.
```
Comment (agallou) [inline source/en/asserters/extension.inc.rst]: the usage is not the same so, something like : "if you need to run tests only if an extension is present, you can use the PHP annotation". Like that, readers will see the diffrence.

File: source/fr/asserters/extension.inc.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (agallou) [inline source/fr/asserters/extension.inc.rst]: the french translation is in english.

**Comment by Grummfy** [comment]
thanks for the review

File: source/en/asserters/extension.inc.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+.. _extension-anchor:
+
+extension
+*********
+
+It's the assertion dedicated to PHP extension.
```
Comment (mikaelrandy) [inline source/en/asserters/extension.inc.rst]: `assertion` or `asserter` ?

File: source/en/asserters/extension.inc.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+.. _extension-is-loaded:
+
+isLoaded
+========
+
+Test if the extension is loaded.
```
Comment (mikaelrandy) [inline source/en/asserters/extension.inc.rst]: "Check if the extension is loaded (installed and enabled)"

File: source/en/asserters/extension.inc.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+.. code-block:: php
+
+	<?php
+	$this
+		->extension('json')
+		->isLoaded()
```
Comment (mikaelrandy) [inline source/en/asserters/extension.inc.rst]: Maybe another indent level

File: source/fr/asserters/extension.inc.rst [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+.. _extension-anchor:
+
+extension
+*********
+
+C'est l'assertion dédiée aux extension PHP.
```
Comment (mikaelrandy) [inline source/fr/asserters/extension.inc.rst]: Same comment as EN

Comment (mikaelrandy) [inline source/fr/asserters/extension.inc.rst]: "aux extensions"

File: source/fr/asserters/extension.inc.rst [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+.. code-block:: php
+
+	<?php
+	$this
+		->extension('json')
+		->isLoaded()
+		;
+
+.. note::
+	Si vous devez exécuté certains tests uniquement dans le cas où une extension est présente, vous pouvez utiliser :ref:`l'annotation PHP<annotation-php-extension>`.
```
Comment (mikaelrandy) [inline source/fr/asserters/extension.inc.rst]: *exécuter

Comment (mikaelrandy) [inline source/fr/asserters/extension.inc.rst]: "l'annotation PHP @extension"

File: source/en/asserters/extension.inc.rst [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+		;
+
+.. note::
+	If you need to run tests only if an extension is present, you can use the :ref:`PHP annotation<annotation-php-extension>`.
```
Comment (mikaelrandy) [inline source/en/asserters/extension.inc.rst]: "the @extension PHP annotation"

**Pull request merged**